GOVERNANCE.md

```diff
@@ -18,15 +18,61 @@ This project may include the following roles. Additional roles may be adopted an
 
 ## 3. How We Work.
 
-**3.1. Openness**. Participation is open to anyone who is directly and materially affected by the activity in question. There shall be no undue financial barriers to participation.
+### 3.1. Principles
 
-**3.2. Balance**. The development process should balance the interests of Contributors and other stakeholders. Contributors from diverse interest categories shall be sought with the objective of achieving balance.
+**3.1.1. Openness**. Participation is open to anyone who is directly and materially affected by the activity in question. There shall be no undue financial barriers to participation. In that sense, discussions should prioritize public channels (Github discussions/issues/PRs or Discord) whenever possible.
```

Comment:
Maybe some comment here also about record keeping and discoverability of past decision making. I think discord is inferior to github for technical discussions for this reason: it's going to be hard for anyone to locate the discussions later. Of course there are tradeoffs here, for instance having a video meeting is very efficient for technical discussions but it is often not very inclusive and if nobody takes meeting minutes then there will be no record of what happened.